docs/configure.md

```diff
@@ -48,12 +48,12 @@ EDOT Python uses different defaults than OpenTelemetry Python for the following
 
 | Option | EDOT Python default | OpenTelemetry Python default |
 |---|---|---|
-| `OTEL_EXPERIMENTAL_RESOURCE_DETECTORS` | `process_runtime,os,otel,telemetry_distro,_gcp,aws_ec2,aws_ecs,aws_elastic_beanstalk,azure_app_service,azure_vm` | `otel` |
+| `OTEL_EXPERIMENTAL_RESOURCE_DETECTORS` | `process_runtime,os,otel,telemetry_distro,service_instance,_gcp,aws_ec2,aws_ecs,aws_elastic_beanstalk,azure_app_service,azure_vm` | `otel` |
```

Comment:
FWIW, the OTel JS string used in the `OTEL_NODE_RESOURCE_DETECTORS` envvar for the "detector" that adds `service.instance.id` is named `"serviceinstance"`.  Anyway, just an FYI. It certainly isn't the job of this PR to look at cross-language commonality,  and there are already many other diffs here.

Reply:
The plan for python is that when these attributes are made stable they would be added to the `otel` resource detector.